strawberry/enum.py

```diff
@@ -41,7 +41,7 @@ def __hash__(self) -> int:
     def copy_with(
         self, type_var_map: Mapping[TypeVar, Union[StrawberryType, type]]
     ) -> Union[StrawberryType, type]:
-        return super().copy_with(type_var_map)
+        return super().copy_with(type_var_map)  # type: ignore[safe-super]
```

Comment:
@BryceBeagle do we need these copy_with? They don't do anything that super doesn't do 🤔

Reply:
I'll merge this PR, but I'm assuming we can remove this code at some point :)